evap/evaluation/templates/base.html

```diff
@@ -213,6 +213,18 @@
             import { setupForms } from "{% static 'js/custom-success-form.js' %}";
 
             setupForms();
+            document.querySelectorAll("[data-track-tomselect-count]").forEach(trackerElement => {
+                let trackedElement = document.getElementById(trackerElement.dataset.trackTomselectCount).tomselect;
+                trackedElement.on("item_add", function() {
+                    update_count(trackerElement, trackedElement);
+                });
+                trackedElement.on("item_remove", function() {
+                    update_count(trackerElement, trackedElement);
+                });
+                function update_count(trackerElement, trackedElement) {
+                    trackerElement.innerText = trackedElement.items.length;
+                }
+            });
```

Comment:
I think we like it more if this is written in a typescript function and executed like `setupForms`